Index: src/test/java/org/example/MapController.java
===================================================================
diff --git a/src/test/java/org/example/MapController.java b/src/test/java/org/example/MapController.java
deleted file mode 100644
--- a/src/test/java/org/example/MapController.java	(revision aa88fba8627bea3bb38c41d6b9e59ee909474669)
+++ /dev/null	(revision aa88fba8627bea3bb38c41d6b9e59ee909474669)
@@ -1,4 +0,0 @@
-package org.example;
-
-public class MapController {
-}
Index: src/test/java/org/example/viewer/MapViewerTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/org/example/viewer/MapViewerTest.java b/src/test/java/org/example/viewer/MapViewerTest.java
new file mode 100644
--- /dev/null	(date 1701043318747)
+++ b/src/test/java/org/example/viewer/MapViewerTest.java	(date 1701043318747)
@@ -0,0 +1,4 @@
+package org.example.viewer;
+
+public class MapViewerTest {
+}
Index: src/test/java/org/example/model/PositionTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/org/example/model/PositionTest.java b/src/test/java/org/example/model/PositionTest.java
new file mode 100644
--- /dev/null	(date 1701043318751)
+++ b/src/test/java/org/example/model/PositionTest.java	(date 1701043318751)
@@ -0,0 +1,4 @@
+package org.example.model;
+
+public class PositionTest {
+}
Index: src/test/java/org/example/viewer/PlayerTankTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/org/example/viewer/PlayerTankTest.java b/src/test/java/org/example/viewer/PlayerTankTest.java
new file mode 100644
--- /dev/null	(date 1701043318743)
+++ b/src/test/java/org/example/viewer/PlayerTankTest.java	(date 1701043318743)
@@ -0,0 +1,4 @@
+package org.example.viewer;
+
+public class PlayerTankTest {
+}
Index: src/test/java/org/example/controllers/NPCSControllerTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/org/example/controllers/NPCSControllerTest.java b/src/test/java/org/example/controllers/NPCSControllerTest.java
new file mode 100644
--- /dev/null	(date 1701043318755)
+++ b/src/test/java/org/example/controllers/NPCSControllerTest.java	(date 1701043318755)
@@ -0,0 +1,4 @@
+package org.example.controllers;
+
+public class NPCSControllerTest {
+}
Index: src/test/java/org/example/GUI/LanternaGUITest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/org/example/GUI/LanternaGUITest.java b/src/test/java/org/example/GUI/LanternaGUITest.java
new file mode 100644
--- /dev/null	(date 1701043318762)
+++ b/src/test/java/org/example/GUI/LanternaGUITest.java	(date 1701043318762)
@@ -0,0 +1,4 @@
+package org.example.GUI;
+
+public class LanternaGUITest {
+}
Index: src/test/java/org/example/controllers/PlayerTankControllerTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/org/example/controllers/PlayerTankControllerTest.java b/src/test/java/org/example/controllers/PlayerTankControllerTest.java
new file mode 100644
--- /dev/null	(date 1701043318758)
+++ b/src/test/java/org/example/controllers/PlayerTankControllerTest.java	(date 1701043318758)
@@ -0,0 +1,4 @@
+package org.example.controllers;
+
+public class PlayerTankControllerTest {
+}
Index: src/test/java/org/example/controllers/MapControllerTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/org/example/controllers/MapControllerTest.java b/src/test/java/org/example/controllers/MapControllerTest.java
new file mode 100644
--- /dev/null	(date 1701043143490)
+++ b/src/test/java/org/example/controllers/MapControllerTest.java	(date 1701043143490)
@@ -0,0 +1,4 @@
+package org.example.controllers;
+
+public class MapControllerTest {
+}
